cli/test/test-diff-utils.mjs

```
@@ -0,0 +1,107 @@
+    await fn()
+    console.log(`✅ PASSED: ${name}`)
+    testsPassed++
+  }
+  catch (error) {
+    console.error(`❌ FAILED: ${name}`)
+    console.error(`   Error: ${error.message}`)
+    testsFailed++
+  }
+}
+
+function assert(condition, message) {
```

ESLint

[error] 'assert' is defined but never used.

(no-unused-vars)